### PR TITLE
[13.0][IMP] sale_report_delivered_subtotal: Add delivered quantity on sale_report

### PR DESCRIPTION
**Pull request by CarlosRoca13**
cc @Tecnativa TT31881

please @sergio-teruel @chienandalu review this

### CODE REVIEW

**Review by chienandalu — APPROVED** [review]

**Review by sergio-teruel — APPROVED** [review]

File: sale_report_delivered_subtotal/reports/sale_report.py [hunk 1]
```diff
@@ -8,17 +8,27 @@ class SaleReport(models.Model):
...
+        select_str_price = """ ,
             sum((l.price_subtotal /
                  coalesce(nullif(l.product_uom_qty, 0), 1)
                 ) * l.qty_delivered)
             as price_subtotal_delivered
         """
-        fields.update({"price_subtotal_delivered": select_str})
+        select_str_weight = """ ,
+            sum(p.weight * l.qty_delivered /
+                u.factor * u2.factor) as weight_delivered
```
Comment (pedrobaeza) [inline sale_report_delivered_subtotal/reports/sale_report.py:24]: This is not quantity, so you shouldn't use UoM factors.

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

File: sale_report_delivered_subtotal/reports/sale_report.py (hunk 1 above)
Comment (chienandalu) [reply sale_report_delivered_subtotal/reports/sale_report.py:24]: ~Right it's already converted:~ https://github.com/odoo/odoo/blob/13.0/addons/sale/report/sale_report.py#L69

Edit:

Sure, you're right (maybe a couple of parenthesis would give more clarity)

Comment (CarlosRoca13) [reply sale_report_delivered_subtotal/reports/sale_report.py:24]: I get this formula from this reference https://github.com/OCA/OCB/blob/25d0db569e26403777918ff179fe6d9993a694d5/addons/sale/report/sale_report.py#L95 changing the `product_uom_qty` by the `qty_delivered`

**Review by pedrobaeza — APPROVED** [review]
Yeah, sorry, it's alright.

**Pull request merged**